apertium-tur-uzb.tur-uzb.dix

```diff
@@ -205,26 +207,31 @@
 <e><p><l>masumane<s n="adv"/></l><r>aybsiz<s n="adv"/></r></p></e>
 <e><p><l>zor<s n="adv"/></l><r>arang<s n="adv"/></r></p></e>
 <e><p><l>en<s n="adv"/></l><r>eng<s n="adv"/></r></p></e>
-
+<e><p><l>çok<s n="adv"/></l><r>juda<s n="adv"/></r></p></e>
 <e><p><l>çok<s n="adv"/></l><r>koʻp<s n="adv"/></r></p></e>
 <e><p><l>daha<s n="adv"/></l><r>battar<s n="adv"/></r></p></e>
 <e><p><l>daha<s n="adv"/></l><r>badtar<s n="adv"/></r></p></e>
 <e><p><l>ardından<s n="adv"/></l><r>axtarish<s n="adv"/></r></p></e>
 
-<e><p><l>en<b/>azından<s n="adv"/></l><r>kamida<s n="adv"/></r></p></e>
+<e><p><l>en<b/>azından<s n="adv"/></l><r>kamida<s n="adv"/></r></p></e>   <!-- Will be moved to MWE later-->
 <e><p><l>ileri<s n="adv"/></l><r>oldinga<s n="adv"/></r></p></e>
 <e><p><l>anca<s n="adv"/></l><r>uncha<s n="adv"/></r></p></e>
-<e><p><l>bir<b/>daha<s n="adv"/></l><r>qaytadan<s n="adv"/></r></p></e>
+<e><p><l>bir<b/>daha<s n="adv"/></l><r>qaytadan<s n="adv"/></r></p></e>   <!-- Will be moved to MWE later-->
 <e><p><l>görünüşte<s n="adv"/></l><r>chogʻi<s n="adv"/></r></p></e>
 <e><p><l>sonra<s n="adv"/></l><r>oʻtgach<s n="adv"/></r></p></e>
 <e><p><l>evvela<s n="adv"/></l><r>avvalo<s n="adv"/></r></p></e>
-<e><p><l>aşağıda<b/>belirtildiği<b/>gibi<s n="adv"/></l><r>quyidagicha<s n="adv"/></r></p></e>
+<e><p><l>aşağıda<b/>belirtildiği<b/>gibi<s n="adv"/></l><r>quyidagicha<s n="adv"/></r></p></e>  <!-- Will be moved to MWE later-->
 <e><p><l>çok<s n="adv"/></l><r>zab<s n="adv"/></r></p></e>
 <e><p><l>sonra<s n="adv"/></l><r>keyinchalik<s n="adv"/></r></p></e>
 <e><p><l>sonra<s n="adv"/></l><r>soʻng<s n="adv"/></r></p></e>
 <e><p><l>geçici<s n="adv"/></l><r>vaqtincha<s n="adv"/></r></p></e>
 <e><p><l>teferruatla<s n="adv"/></l><r>batafsil<s n="adv"/></r></p></e>
-
+<e><p><l>çabukça<s n="adv"/></l><r>zudlik<b/>bilan<s n="adv"/></r></p></e> <!-- Will be moved to MWE later-->
+<e><p><l>dün<s n="adv"/></l><r>kecha<s n="adv"/></r></p></e>
+<e><p><l>ora<s n="adv"/></l><r>u% yer<s n="adv"/></r></p></e>
+<e><p><l>bura<s n="adv"/></l><r>bu% yer<s n="adv"/></r></p></e>
+<e><p><l>şura<s n="adv"/></l><r>shu% yer<s n="adv"/></r></p></e>
```

Comment:
Spaces are with `<b/>`.  Also, these should be moved to `lsx` when that's possible.

Reply:
Ah, my bad again, fixed. I just keep forgetting to replace those spaces :)